### PR TITLE
fix: add adornment to setting

### PR DESCRIPTION
**Pull request by solaolateju**
**Added inner textfield seconds**
Screenshot 2020-04-17 at 18 59 15

**Hour hidden when start time is less than an hour**
Screenshot 2020-04-18 at 00 57 17

**Removed days from student view**
Screenshot 2020-04-17 at 18 58 50

### CODE REVIEW

File: src/components/modes/student/StudentView.js [hunk 1]
```diff
@@ -32,27 +32,32 @@ const useStyles = makeStyles(theme => ({
 export const StudentView = props => {
   const { t, initialTimeValue, direction, tool } = props;
   const classes = useStyles();
+  const MillSecsToMin = 60;
 
   return (
     <div>
       <Timer
-        initialTime={initialTimeValue}
+        initialTime={initialTimeValue * 60000}
```
Comment (juancarlosfarah) [inline src/components/modes/student/StudentView.js:40]: Factor this out to a constant `ONE_MINUTE_IN_MILLIS`.

File: src/components/modes/student/StudentView.js [hunk 2]
```diff
@@ -32,27 +32,32 @@ const useStyles = makeStyles(theme => ({
 export const StudentView = props => {
   const { t, initialTimeValue, direction, tool } = props;
   const classes = useStyles();
+  const MillSecsToMin = 60;
```
Comment (juancarlosfarah) [inline src/components/modes/student/StudentView.js:35]: This is hours in min, not millis. Double check that it's right and use the right name in all caps. `ONE_HOUR_IN_MINS`.

File: src/components/modes/student/StudentView.js [hunk 3]
```diff
@@ -32,27 +32,32 @@ const useStyles = makeStyles(theme => ({
...
                 <Grid
                   className={classes.gridRow}
                   container
                   spacing={3}
                   direction={tool ? 'column' : 'row'}
                   alignItems="center"
                 >
-                  <Counter timeValue={<Timer.Days />} timeUnit={t('Days')} />
-                  <Counter timeValue={<Timer.Hours />} timeUnit={t('Hours')} />
+                  {initialTimeValue > MillSecsToMin && (
```
Comment (juancarlosfarah) [inline src/components/modes/student/StudentView.js:55]: This should be `>=`

File: src/components/modes/teacher/Settings.js [hunk 4]
```diff
@@ -138,11 +147,18 @@ class Settings extends Component {
           label={t('Show Header to Students')}
         />
         <TextField
-          className={classes.formControl}
           type="Number"
           value={initialTimeValue}
+          id="outlined-start-adornment"
```
Comment (juancarlosfarah) [inline src/components/modes/teacher/Settings.js:152]: change this to `initialTimeValue`

File: src/components/modes/teacher/Settings.js [hunk 5]
```diff
@@ -149,10 +157,12 @@ class Settings extends Component {
         <TextField
           type="Number"
           value={initialTimeValue}
-          id="outlined-start-adornment"
+          id="initaiTimeValue"
```
Comment (juancarlosfarah) [inline src/components/modes/teacher/Settings.js:160]: Typo. Please be careful with these types of errors.

File: src/components/modes/teacher/Settings.js [hunk 6]
```diff
@@ -116,15 +117,22 @@ class Settings extends Component {
   };
 
   handleChangeInitialTimeValue = ({ target: { value } }) => {
-    this.setState({
-      initialTimeValue: Number(value),
-    });
+    if (Number(value) < 0) {
```
Comment (juancarlosfarah) [inline src/components/modes/teacher/Settings.js:120]: Does this work correctly? If they start typing `-`, what shows?

Comment (juancarlosfarah) [reply src/components/modes/teacher/Settings.js:120]: Maybe reset it to 0 if it is under 0?

Comment (solaolateju) [reply src/components/modes/teacher/Settings.js:120]: Error State - (It currently do not allow negative numbers)
Screenshot 2020-04-23 at 13 42 42

Default State
Screenshot 2020-04-23 at 13 42 03

**Pull request merged**